src/ops/mod.rs

```diff
@@ -340,7 +340,7 @@ impl GitRepoPackage {
             GitRepoPackage {
                 name: c.get(1).unwrap().as_str().to_string(),
                 url: url.into_string(),
-                branch: branch,
+                branch,
```

Comment:
I'd actually rather this stay explicit, as it's more aesthetically pleasing, if you please.

Reply:
I'll revert this change and disable the lint.